Project.toml

```
@@ -4,20 +4,18 @@ authors = ["Beacon Biosignals, Inc."]
 version = "0.1.2"
```

I think we should probably mark this as breaking; I suspect it could break folks binary installation setup:
```suggestion
version = "0.2.0"
```